README.md

```diff
@@ -180,10 +171,10 @@ Thanks a bunch for wanting to help Zooniverse. Here are few quick guidelines to
 0. Submit a Pull Request
 0. Wait for feedback or a merge!
 
-Your Pull Request will run on [travis-ci](https://travis-ci.org/zooniverse/Panoptes), and we'll probably wait for it to pass on MRI Ruby 2.4. For more information, [see the wiki](https://github.com/zooniverse/Panoptes/wiki/Contributing-to-Panoptes).
+Your Pull Request will run on via github actions. For more information, [see the wiki](https://github.com/zooniverse/Panoptes/wiki/Contributing-to-Panoptes).
```

Comment:
```suggestion
Your Pull Request will run via github actions. For more information, [see the wiki](https://github.com/zooniverse/Panoptes/wiki/Contributing-to-Panoptes).
```

Also I clicked this link, and I dont think this page exists https://github.com/zooniverse/Panoptes/wiki/Contributing-to-Panoptes. We probably meant this page : https://github.com/zooniverse/panoptes/wiki/Development-Guide-for-Panoptes ?